CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [3.0.0] - 2019-02-28
+
+### Added
+- Added `automatic_software_updates` resource to enable or disable the automatic checking, downloading, and installing of software updates.
+- Added `azure-pipelines.yml` to allow for managing builds as code.
+- Added some resource unit tests for `spotlight` to complement the existing `metadata_util` tests.
+
+### Changed
+- Changed the `ard` resource to `remote_management` and updates applicable tests and documentation. The new `remote_management` resource greatly simplifies syntax and reduces the needed macOS domain knowledge around `kickstart` options. However, it has less functionality than `ard` and is a significant breaking change.
+
+### Fixed
+- Fixed .mailmap file to accurately track contributor emails.
+- Fixed guard in the `keychain` resource for the `:create` action.
+
+### Removed
+- Officially removed support for Chef 13 and macOS El Capitan.
```

Comment:
As opposed to _Un_-officially removing support for these?

Reply:
Would you prefer "finally removed"? Its a reference to our deprecation notice in a previous release.